Copilot for Xcode/OpenAIView.swift

```diff
@@ -30,64 +30,43 @@ struct OpenAIView: View {
 
                 Form {
                     HStack {
-                        Text("OpenAI API Key")
                         TextField(text: $settings.openAIAPIKey, prompt: Text("sk-*")) {
-                            EmptyView()
-                        }.textFieldStyle(.copilot)
+                            Text("OpenAI API Key")
+                        }.textFieldStyle(.roundedBorder)
                         Button(action: {
                             openURL(apiKeyURL)
                         }) {
                             Image(systemName: "questionmark.circle.fill")
-                        }
-                        .buttonStyle(.plain)
+                        }.buttonStyle(.plain)
                     }
 
                     HStack {
-                        Text("ChatGPT Model")
-                        TextField(text: $settings.chatGPTModel, prompt: Text("gpt-3.5-turbo")) {
-                            EmptyView()
-                        }.textFieldStyle(.copilot)
-
+                        Picker(selection: $settings.chatGPTModel) {
+                            ForEach(ChatGPTModel.allCases, id: \.self) { model in
+                                Text(model.rawValue).tag(model.rawValue)
+                            }
+                        } label: {
+                            Text("ChatGPT Model")
+                        }.pickerStyle(.menu)
                         Button(action: {
                             openURL(modelURL)
                         }) {
                             Image(systemName: "questionmark.circle.fill")
-                        }
-                        .buttonStyle(.plain)
+                        }.buttonStyle(.plain)
                     }
-
-                    HStack {
-                        Text("ChatGPT Endpoint")
-                        TextField(
-                            text: $settings.chatGPTEndpoint,
-                            prompt: Text("https://api.openai.com/v1/chat/completions")
-                        ) {
-                            EmptyView()
-                        }.textFieldStyle(.copilot)
+                    .onChange(of: settings.chatGPTModel) { newValue in
+                        guard let model = ChatGPTModel(rawValue: newValue) else { return }
+                        settings.chatGPTEndpoint = model.endpoint
+                        settings.chatGPTMaxToken = model.maxToken
```

Comment:
Where can the user select the token size? Do we need to bring the textfield back?

Reply:
Allow users to change token size
Implemented with stepper because of the risk of non-numeric input in text fields

What do you think?

![スクリーンショット 2023-03-29 9 24 55](https://user-images.githubusercontent.com/84154073/228395991-d4a422a7-c271-45f2-bd6a-2285374956a3.png)

Reply:
You can use the `format` parameter to `TextField` to constrain the input to a number format, see https://github.com/j-f1/GPT-3-Playground/blob/main/GPT-3%20Playground/ConfigView.swift for an example.

Reply:
MaxToken is preconfigured in the language model.
If you make it a text field, users may accidentally enter more than 4,096

I consider this a bad UX.

I would like to prevent k this using Sptepper or Slider etc

![スクリーンショット 2023-03-29 10 30 32](https://user-images.githubusercontent.com/84154073/228403184-d04cee8a-d658-4ae0-b806-d2c2cf867e0c.png)

Reply:
I think it’s fine to let people enter whatever they want (with perhaps a warning if >4096) since future models may have a higher limit.

Reply:
Indeed it is.
I'll change it back to a text field.